Src/BootCamp.Chapter/Demo.cs

```diff
@@ -0,0 +1,133 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace BootCamp.Chapter
+{
+
+    public class Demo
+    {
+        Dictionary<string, ConsoleKey> consoleOptions = new Dictionary<string, ConsoleKey>();
+        public event LogEventHandler logEventHandler;
```

Comment:
Should be renamed to ``OnMenuKeyPressed``

Reply:
And in general should belong to the menu class. inside demo, you can demo how you handle that even. But it's good enough for this homework.

Reply:
Do I understand correctly that I should make a new class for the menu and put the login in there? Sounds better. I'll see if I can get that fixed.

Reply:
Yes

Reply:
much cleaner. thank you.